### ISSUE TITLE
Wrong fix of new without parentheses on anonymous readonly class

### ISSUE THREAD

**Issue by paranoiq** [slevomat/coding-standard#1673]
rule: Usage of "new" without parentheses is disallowed. (SlevomatCodingStandard.ControlStructures.NewWithParentheses.MissingParentheses)

```
$object3 = new class {
    public readonly int $foo;
    public int $bar;
};
```
is not detected at all (this may be a feature? - running with default configuration: `<rule ref="SlevomatCodingStandard.ControlStructures.NewWithParentheses"/>`)

```
$object4 = new readonly class {
    public int $foo;
};
```
is autofixed to invalid code (wrong parentheses placement):
```
$object4 = new readonly class {
    public int $foo();
};
```

this is ok (no error. not changed):
```
$object4 = new readonly class() {
    public int $foo;
};
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/5cc68dc1d74024003adaaefbfea8c46d6fbe6191
